npm-update

```diff
@@ -94,6 +94,9 @@ def run(specified_package, verbose=False, **kwargs):
                     if task.verbose:
                         sys.stderr.write("Ignoring '{0}' as there is pending PR #{1}\n".format(pkg, issue["number"]))
 
+    # The only time git will checkout an empty subdir is if it's a gitlink
+    is_gitlink = not subprocess.call(['rmdir', 'node_modules'])
```

Comment:
Seems a bit strange to not use [os.rmdir()](https://docs.python.org/3/library/os.html#os.rmdir) instead (and catch `OSError`), but sure.

Reply:
Eh.  I didn't feel like dealing with the try/catch nonsense and also avoiding a blanket catch by catching both the "Directory is full" and "Directory doesn't exist" cases... etc etc.  I started coding that, and it sucked.

I also tried to query the information directly out of git but that's not so elegant either.

I think this is fine.